nb2workflow/nbadapter.py

```diff
@@ -540,6 +549,33 @@ def extract_output_declarations(self):
     def extract_output(self):
         return self.extract_pm_output()
 
+    def download_local_files(self, parameters, tmpdir):
+        adapted_parameters = copy.deepcopy(parameters)
+        issues = []
+        for input_par_name, input_par_obj in self.input_parameters.items():
+            if input_par_obj['owl_type'] == "http://odahub.io/ontology#POSIXPath":
```

Comment:
For the future, it may be better to analyse the ontology using `oda_api.ontology_helper` so that this also works for the subclasses. 
May be extended later.